hudi-cli/src/test/java/org/apache/hudi/cli/integ/ITTestCleansCommand.java

```diff
@@ -1,106 +0,0 @@
-/*
```

Comment:
@yanghua : I went ahead and removed this class. I have added a jira : https://issues.apache.org/jira/browse/HUDI-1033 to go through other such examples and see if multiple CLI tests are still needed.

Reply:
Hi @bvaradar replied you on the jira.